### ISSUE TITLE
docker: label matching not working with multiple labels

### ISSUE THREAD

**Issue by amitsaha** [inspec/inspec#3653]
### 🎛 Description

🐛 bug report

Label matching doesn't seem to work correctly when we have multiple labels. It seems like `include` is doing an exact match? Please see below.

### 🌍 InSpec and Platform Version

OS: Linux
```

$ inspec version
3.1.3
```
### 🤔 Replication Case

Test:

```
describe docker.containers.where { names == "amazing_noether" } do
    its('labels') { should include 'version=foo' }
end
```

Run an example container:

```
$ sudo docker run --label="app=redis" --label="version=foo" -d redis
```
Run test:

```
 sudo inspec  exec test.rb

Profile: tests from test.rb (tests from test.rb)
Version: (not specified)
Target:  local://

  #<Inspec::Resources::DockerContainerFilter:0x000000000587aac8> with names == "amazing_noether"
     ×  labels should include "version=foo"
     expected ["app=redis,version=foo"] to include "version=foo"

Test Summary: 0 successful, 1 failure, 0 skipped
```
The issue seems to be that `["app=redis, version=foo"]` should really be `["app=redis", "version=foo"]`. I tried my hand at fixing this up, but not yet been successful.

### 💁 Possible Solutions

Not sure.

### 💻 Stacktrace

NA

**Comment by lain0** [comment]
try to update inspec, this works fine vs inspec 3.1.3

```
"Labels": {
    "com.docker.compose.project": "docker-compose-letsencrypt-nginx-proxy-companion",
    "com.github.jrcs.letsencrypt_nginx_proxy_companion.nginx_proxy": "true",
    "maintainer": "NGINX Docker Maintainers <docker-maint@nginx.com>"
},
```
```
describe docker_container('nginx-web') do
  its('labels') {should include 'com.github.jrcs.letsencrypt_nginx_proxy_companion.nginx_proxy=true' }
end
```

**Comment by amitsaha** [comment]
thanks for that @lain0 - `docker_container` works for me.

However, I see the same result with my original issue where I use `describe docker.containers.where `.

**Comment by amitsaha** [comment]
Example `docker ps`:

```
$ sudo docker ps -a --format '{{json . }}'
{"Command":"\"docker-entrypoint.s…\"","CreatedAt":"2018-12-17 03:52:18 +0000 UTC","ID":"a614de84f3d3","Image":"redis","Labels":"app=redis,version=foo","LocalVolumes":"1","Mounts":"5c10bbb910db94…","Names":"nostalgic_lichterman","Networks":"bridge","Ports":"6379/tcp","RunningFor":"20 hours ago","Size":"0B","Status":"Up 20 hours"}
```

**Comment by amitsaha** [comment]
Thanks to @jerryaldrichiii for helping on this and trying to fix it up in #3673 . For now, this alternative approach works (with the fix):

```
containers = docker.containers.where { labels.include?('version=foo') }.entries
containers.each do |container|
  describe docker_container(id: container.id) do
    its('labels') { should include 'app=redis' }
    it { should be_running }
  end
end
```
